MergerLogic/Monitoring/Metrics/IMetricsProvider.cs

```diff
@@ -7,7 +7,6 @@ namespace MergerLogic.Monitoring.Metrics
     public interface IMetricsProvider
     {
         void TaskExecutionTimeHistogram(double measuredTime, string taskType);
-        void BatchInitializationTimeHistogram(double measuredTime);
```

Comment:
Please restore this, we still want this information to see the amount of time it takes for the data to build.
Maybe it's name should change because we moved the code.